internal/app/machined/pkg/system/health/health_test.go

```diff
@@ -93,7 +93,12 @@ func (suite *CheckSuite) TestHealthChange() {
 		errCh <- health.Run(ctx, &settings, &state, check)
 	}()
 
-	time.Sleep(50 * time.Millisecond)
+	for i := 0; i < 10; i++ {
+		time.Sleep(20 * time.Millisecond)
+		if *state != nil {
```

Comment:
this should be state.Get().Healthy != nil, but I need to look deeper to see if it's correct

Reply:
`state.Get()` returns `nil` until first healthcheck runs 

accessing `*state` is illegal cause it's not locked properly